README.md

```diff
@@ -1,3 +1,92 @@
+
+
+## Proxy Tester
+
+The Proxy Tester script is designed to test the validity and response time of HTTP/HTTPS proxies. It uses the `faster_than_requests` library for faster proxy testing.
```

Comment:
The code does not seem like you are using faster_than_requests library, rather you are using concurrent threads for making requests which is not an issue but update the README accordingly